data/ecosystems/s/stacks.toml

```diff
@@ -29,6 +29,16 @@ github_organizations = [
   "https://github.com/runkod",
   "https://github.com/SpringRole",
   "https://github.com/StackedSats",
+  "https://github.com/Trust-Machines",
+  "https://github.com/secretkeylabs",
+  "https://github.com/Clarity-Innovation-Lab",
+  "https://github.com/fungible-systems",
+  "https://github.com/mechanismHQ",
+  "https://github.com/stacksfoundation",
+  "https://github.com/magicstx",
+  "https://github.com/lnswap",
+  "https://github.com/layerswap",
```

Comment:
Most of these look like Stacks orgs except for layerswap.  Can you remove this one?

Also if possible, many of these orgs look like they could be their sub ecosystems of Stacks or Bitcoin.  If they have a discord / twitter account, that usually is a good heuristic of whether they should be their own ecosystem.

Reply:
Removed Layerswap! Appreciate the fast reviews.

Reply:
@jubos any further feedback? Any blockers to merge this?